wire/internal/wire/wire.go

```diff
@@ -113,7 +117,7 @@ func copyNonInjectorDecls(g *gen, files []*ast.File, info *types.Info) {
 		for _, decl := range f.Decls {
 			switch decl := decl.(type) {
 			case *ast.FuncDecl:
-				if isInjector(info, decl) != nil {
+				if buildCall, _ := isInjector(info, decl); buildCall != nil {
```

Comment:
It's not obvious why the error can be ignored: add a comment that any error cases would already be filtered out.

Reply:
Done.